i18n/en.toml

```diff
@@ -61,6 +61,9 @@ other = "Yes"
 [input_placeholder_email_address]
 other = "email address"
 
+[subscribe_button]
+other = "Subscribe"
+
```

Comment:
This file is sorted alphabetically, I think. Would it be OK to move this so the file stays sorted @Arhell?

Reply:
done, thanks